netstandard/ref/System.Text.RegularExpressions.cs

```diff
@@ -71,7 +129,6 @@ public partial class Regex : System.Runtime.Serialization.ISerializable
         protected internal string[] capslist;
         protected internal System.Text.RegularExpressions.RegexRunnerFactory factory;
         public static readonly System.TimeSpan InfiniteMatchTimeout;
-        [System.Runtime.Serialization.OptionalFieldAttribute(VersionAdded=2)]
```

Comment:
@ViktorHofer isn't this a potential serialization breaking change? I couldn't find this optional field attribute in our .NET Core implementation but it would seem like that would cause breaks for serialization round-tripping. 

Note that I don't think this is a break for the reference assembly but for the runtime assembly.

Reply:
This is a breaking change indeed and should be undone. If blobs are exchanged with very old implementations, i.e. with netfx that don't have this field this will throw.

Reply:
As pointed out below, that's about the implementation only.